gradle.properties

```diff
@@ -1,5 +1,5 @@
 #Sat, 15 Aug 2015 15:15:18 +0000
-version=1.10.1-SNAPSHOT
+version=1.11.0-SNAPSHOT
```

Comment:
because the thrift service definition change. eventhough we soon will move this out of thrift.